cypress/e2e/dashboard_atomic_tests.cy.js

```diff
@@ -0,0 +1,185 @@
+    });
+
+    it("Support tab is visible", () => {
+      new HomePage().supportMenu.isVisibleXpath();
+    });
+
+    it("Support tab has text", () => {
+      new HomePage().supportMenu.hasTextXpath("Podpora");
+    });
+  });
+
+  context("Footer tests", () => {
+    it("Footer is visible", () => {
+      new HomePage().footer.isVisible();
+    });
+
+    it("Footer have text", () => {
+      new HomePage().footer.hasText("© 2023 Banking App");
+    });
+  });
+
+  context("Profile details on Dashboard tests", () => {
+    it("Profile Details header is visible", () => {
+      new HomePage().profileDetailHeader.isVisible();
+    });
+
+    it("Profile Details header has text", () => {
+      new HomePage().profileDetailHeader.hasText("Detaily Profilu");
+    });
+
+    it("Edit Profile Buttion is visible", () => {
+      new HomePage().editProfileButton.isVisible();
+    });
+
+    it("Edit Profile Button has text", () => {
+      new HomePage().editProfileButton.hasText("Upravit profil");
+    });
+
+    it("First Name has attribute", () => {
+      new HomePage().firstName.hasAttribute("data-testid", "name");
+    });
+
+    it("Last Name has attribute", () => {
+      new HomePage().lastName.hasAttribute("data-testid", "surname");
+    });
+
+    it("Email has attribute", () => {
+      new HomePage().email.hasAttribute("data-testid", "email");
+    });
+
+    it("Phone Number has attribute", () => {
+      new HomePage().phoneNumber.hasAttribute("data-testid", "phone");
+    });
+
+    it("Age has attribute", () => {
+      new HomePage().age.hasAttribute("data-testid", "age");
+    });
+  });
+
+  context("Accounts details on Dashboard tests", () => {
+    it("Accounts header is visible", () => {
+      new HomePage().accountsHeader.isVisible();
+    });
+
+    it("Accounts header has text", () => {
+      new HomePage().accountsHeader.hasText("Účty");
+    });
+
+    it("Add account button is visible", () => {
+      new HomePage().addAccountButton.isVisible();
+    });
+
+    it("Account number has attribute", () => {
+      new HomePage().accountNumberHeading.hasAttribute(
+        "data-testid",
+        "account-number-heading"
+      );
+    });
+
+    it("Balance has attribute", () => {
+      new HomePage().accountBalanceHeading.hasAttribute(
+        "data-testid",
+        "account-balance-heading"
+      );
+    });
+
+    it("Account type has attribute", () => {
+      new HomePage().accountTypeHeading.hasAttribute(
+        "data-testid",
+        "account-type-heading"
+      );
+    });
+  });
+  context("Dashboard Buttons functionality tests", () => {
+    it("Edit profile button test", () => {
+      new HomePage()
+        .clickEditProfileButton()
+        .editButton.hasText("Zrušit úpravy");
+    });
+    it("Logout button functionality test", () => {
+      new HomePage().clicklogout().loginPageLogo.isVisible();
+    });
+    /*Add account button testy padají, pokud jsou samostatně (it.only()), tak fungují.
```

Comment:
Testy ti padají kvůli tomu, že tě test `it("Logout button functionality test", () => { ... }` odhlásí. Je super, že je máš aspoň skipnuté. Tak tě nebudou stát více bodů.